From 53a4f98cb871a59df577c99ff9af089c383b4153 Mon Sep 17 00:00:00 2001
From: Frankie Zeng <frankie.zeng@nxp.com>
Date: Fri, 24 Jun 2022 22:01:00 +0800
Subject: [PATCH] remove unused build

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 88de84bf..9c53052f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,9 +13,9 @@ else()
         message(STATUS "The compiler ${CMAKE_CXX_COMPILER} has no C++11 support. Please use a different C++ compiler.")
 endif()
 
-add_subdirectory("thirdparty/cpputest")
-add_subdirectory("daemons/common/tests")
-add_subdirectory("daemons/mrpd")
+#add_subdirectory("thirdparty/cpputest")
+#add_subdirectory("daemons/common/tests")
+#add_subdirectory("daemons/mrpd")
 add_subdirectory("daemons/maap")
 
 message("
